### PR TITLE
allow http header icy-metadata

### PR DESCRIPTION
**Pull request by FernetMenta**
fix http://trac.kodi.tv/ticket/16948

### CODE REVIEW

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by arnova** [comment]
Looks good

File: xbmc/filesystem/CurlFile.cpp [hunk 1]
```diff
@@ -826,13 +827,19 @@ void CCurlFile::ParseAndCorrectUrl(CURL &url2)
...
+                 name == "if-match" || name == "if-modified-since" || name == "if-none-match" ||
+                 name == "if-range" || name == "if-unmodified-since" || name == "max-forwards" ||
+                 name == "origin" || name == "pragma" || name == "range" || name == "te" ||
+                 name == "upgrade" || name == "via" || name == "warning" ||
+                 name == "x-requested-with" || name == "dnt" || name == "x-forwarded-for" ||
+                 name == "x-forwarded-host" || name == "x-forwarded-proto" ||
+                 name == "front-end-https" || name == "x-http-method-override" ||
+                 name == "x-att-deviceid" || name == "x-wap-profile" || name == "x-uidh" ||
+                 name == "x-csrf-token" || name == "x-request-id" || name == "x-correlation-id" ||
+                 name == "icy-metadata")
```
Comment (Sorien) [inline xbmc/filesystem/CurlFile.cpp]: same for https://github.com/xbmc/xbmc/blob/2ec0be0dba35741554d234da2c2cce4d5b62c364/xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp#L670 ?

Comment (FernetMenta) [inline xbmc/filesystem/CurlFile.cpp]: why should ffmpeg want this data?

**Pull request merged**